src/components/ContactUsForm/Contact_Us.css

```diff
@@ -0,0 +1,97 @@
+:root {
+    font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+    color: rgba(255, 255, 255, 0.87);
+    background-color: #242424;

```

Comment:
We wont be needing that as the background color of this form and the original bg-color of the site is same so no need to state explicitly 

we can remove

```
color: rgba(255, 255, 255, 0.87);
background-color: #242424;
```